app/controllers/users_controller.rb

```diff
@@ -19,7 +21,7 @@ def create
   end
 
   def show
-    @user = User.find_by(slug: params[:slug])
+    @user = User.find_by(display_name: params[:display_name])
     @items = Item.active.available.where.not(id: session[:cart]).where(user_id: @user.id)
     @user_events = @user.group_events
```

Comment:
This was the udpate, where now whatever is in the path, we can assume it matches the user's display name instead of having to do the slug conversion